dts/bindings/ipc/nordic,nrf-ipct-common.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nordic IPCT (Interprocessor Communication Transceiver)
```

Comment:
nit: add example snippets, these bindings are complex enough that it would be good to have, also with some extended details on what IPCT does